src/Context/WebApiContext.php

```diff
@@ -241,7 +264,8 @@ public function theResponseShouldNotContain($text)
     public function theResponseShouldContainJson(PyStringNode $jsonString)
     {
         $etalon = json_decode($this->replacePlaceHolder($jsonString->getRaw()), true);
-        $actual = $this->response->json();
+        $actual = $this->response instanceof ResponseInterface
+            ? json_decode($this->response->getBody(), true) : $this->response->json();
```

Comment:
given that Guzzle 4-5 also exposes ``getBody``, you could make the code simpler by always decoding it ourselves.

And you need to validate that the response body was a valid JSON now (``$response->json()`` does it in Guzzle 4-5, but your guzzle 6 code does not do it)